documentation/modules/configuring/proc-resizing-persistent-volumes.adoc

```diff
@@ -20,13 +20,12 @@ Decreasing the size of persistent volumes is not currently supported in Kubernet
 
 .Procedure
 
-. In a `Kafka` resource, increase the size of the persistent volume allocated to the Kafka cluster, the ZooKeeper cluster, or both.
+. Edit a `Kafka` resource using the `size` property to increase the size of the persistent volume allocated to a Kafka cluster, a ZooKeeper cluster, or both.
```

Comment:
`Edit a Kafka resource using the size property` sounds a bit weird to me. Would something like this be better?

```suggestion
. Edit a `Kafka` resource and change the `size` property to increase the size of the persistent volume allocated to a Kafka cluster, a ZooKeeper cluster, or both.
```